src/material/slider/slider.ts

```diff
@@ -503,6 +503,10 @@ export class MatSlider
     const shouldInvertAxis = this._shouldInvertAxis();
     return this._getDirection() == 'rtl' && !this.vertical ? !shouldInvertAxis : shouldInvertAxis;
   }
+  /** It returns safe tick interval percent coercing NaN and Infinity to fallback */
+  private _getTickIntervalPercent(fallback = 0) {
```

Comment:
Instead of defining the getter and having to check that it's a safe number every time, it might be easier to update `_updateTickIntervalPercent` so that `_tickIntervalPercent` is never assigned an invalid number in the first place.

Reply:
Yeah, I didn't somehow notice that `_updateTickIntervalPercent` is the only one place where we assign value to this property. Ok, then it absolutely makes sense. I just pushed refactored code